zulipterminal/model.py

```diff
@@ -962,6 +964,39 @@ def get_other_subscribers_in_stream(
                 if sub != self.user_id
             ]
 
+    def _clean_custom_profile_data(
+        self, custom_profile_data: Dict[str, CustomFieldValue]
+    ) -> Dict[str, CustomProfileData]:
```

Comment:
This could be changed to return a list. This would also make it easier to keep it sorted according to order. Also, we could remove the field_id since we don't use it anymore. I've made the change